pype/lib.py

```diff
@@ -708,3 +709,607 @@ def __init__(self):
     @abstractmethod
     def execute(self, *args, **kwargs):
         pass
+
+
+def get_workfile_build_presets(task_name):
```

Comment:
Somehow this function name makes me assume the result is a list of presets, but looking at the function implementation it seems to return a single `per_task_preset`. Could this benefit from a docstring to describe what it does?